### PR TITLE
Add links to `pluto.md`

### PR DESCRIPTION
**Pull request by TotallyNotChase**
This is meant to be a comprehensive guide for getting started with Pluto.

Please let me know if you spot any mistakes/typos!

### CODE REVIEW

**Review by srid — APPROVED** [review]
Looks detailed and thorough!

File: pluto.md
Comment (srid) [inline pluto.md]: Just noticed this. Could you increase the heading level throughout this file? Look at other `.md` files for comparison. Each `.md` file is expected to use level-2+ headers (and an optional *single* level-1 header at the top).

Comment (TotallyNotChase) [reply pluto.md]: I originally did not want to do this since there's quite a bit heading nesting - like [these](https://github.com/TotallyNotChase/plutonomicon/blob/pluto/pluto.md#integer-constant---42).

**Comment by TotallyNotChase** [comment]
Moved to [pluto](https://github.com/Plutonomicon/pluto/pull/38). This PR now contains updates and links to the pluto guide.

~~Please *only* merge this **after** the pluto guide PR has been merged on Plutonomicon/pluto.~~ It's ready for merge now!

**Pull request merged**
